pattern-2/bosh-release/create.sh

```diff
@@ -162,11 +169,13 @@ cd ..
 echo "---> Adding blobs..."
 
 bosh -e vbox add-blob ${distributions}/${jdk_distribution} openjdk/${jdk_distribution}
-bosh -e vbox add-blob ${distributions}/${mysql_driver} mysqldriver/${mysql_driver}
 bosh -e vbox add-blob ${distributions}/${wso2_product_pack_identifier}.zip ${wso2_product}/${wso2_product_pack_identifier}.zip
 bosh -e vbox add-blob ${distributions}/${wso2_product_keymanager_pack_identifier}.zip wso2is_km/${wso2_product_keymanager_pack_identifier}.zip
 bosh -e vbox add-blob ${distributions}/${wso2_product_analytics_pack_identifier}.zip ${wso2_product}_analytics/${wso2_product_analytics_pack_identifier}.zip
 
```

Comment:
Shall we remove this unnecessary empty line?

Reply:
fixed with https://github.com/wso2/pivotal-cf-apim/pull/49/commits/328ee6d6be3730567554d39a86a2c2829dd01c5c